bindings/Clutter/pkg.info

```diff
@@ -0,0 +1,29 @@
+{
+    "name": "gi-clutter",
+    "version": "1.26.4",
+    "description": "Low-level bindings for clutter, autogenerated by haskell-gi.",
+    "synopsis": "clutter GObject bindings",
+    "girName": "Clutter",
+    "girVersion": "1.0",
```

Comment:
There is now support for an `author` field in `pkg.info`, see https://github.com/haskell-gi/haskell-gi/commit/aa27e772acc94ec55debf411662d5c8a2e4bbbe6 for an example. Could you please add yourself as an author? (I will still appear as the maintainer.)